rust/src/lib/nm/nm_dbus/gen_conf/route_rule.rs

```diff
@@ -23,10 +25,26 @@ impl NmIpRouteRule {
                 keys.push(to_str);
             }
 
-            let from_str = match (self.from.as_ref(), self.from_len.as_ref()) {
-                (Some(f), Some(f_len)) => format!("from {f}/{f_len}"),
-                (Some(f), None) => format!("from {f}"),
-                _ => "".to_string(),
+            let mut from_str =
+                match (self.from.as_ref(), self.from_len.as_ref()) {
+                    (Some(f), Some(f_len)) => format!("from {f}/{f_len}"),
+                    (Some(f), None) => format!("from {f}"),
+                    _ => "".to_string(),
+                };
+            from_str = match (self.from.as_ref(), self.to.as_ref()) {
```

Comment:
Please use `if self.from.as_ref().is_none() && self.to.as_ref().is_none()` instead of match when you only handle single condition.

Reply:
Done, thank you!